src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -311,6 +311,81 @@ public static void IsPathRooted_Windows_Invalid(string value)
         }
 
         [Fact]
+        public static void IsPathFullyQualified_NullThrows()
+        {
+            Assert.Throws<ArgumentNullException>(() => Path.IsPathFullyQualified(null));
+        }
+
+        [Fact]
+        public static void IsPathFullyQualified_Empty()
+        {
+            Assert.False(Path.IsPathFullyQualified(""));
+        }
+
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
This needs conditionalized to not run on desktop (needs [SkipOnTargetFramework(...)]